src/FindStmtCost.h

```diff
@@ -0,0 +1,147 @@
+#ifndef FINDSTMTCOST_H
+#define FINDSTMTCOST_H
+
+#include "Error.h"
+#include "ExternFuncArgument.h"
+#include "Function.h"
+#include "IRVisitor.h"
+#include "Module.h"
+
+#include <stdexcept>
+#include <unordered_map>
+
+using namespace Halide;
```

Comment:
`using namespace` in a .h file is strictly prohibited and should ~never be done.